src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1856,7 +1856,7 @@ internal DeclarationTable Declarations
         /// </summary>
         public override ImmutableArray<Diagnostic> GetParseDiagnostics(CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Design team decided that we should add a new CompilationOption for reporting suppressed diagnostics. So no APIs will need changes. I will revert all of GetXXXDiagnostics API changes in next PR.